src/components/Tooltip/Tooltip.js

```diff
@@ -374,6 +374,11 @@ class Tooltip extends Component {
   handleKeyPress = evt => {
     const key = evt.key || evt.which;
 
+    if (key === 'Escape' || key === 27) {
```

Comment:
```suggestion
    if (key === 'Escape' || key === 'Esc' || key === 27) {
```

Talking with @vpicone revealed that `evt.key` yields `Esc` in IE and looks that we should cover that.

Reply:
would we want to also update and use these helper functions https://github.com/IBM/carbon-components-react/blob/master/src/tools/key.js?

Reply:
@emyarod Good point - I think it makes sense to me.